librad/src/git/storage/fetcher.rs

```diff
@@ -262,6 +262,20 @@ pub mod error {
         #[error(transparent)]
         Pool(#[from] super::PoolError),
     }
+
+    #[derive(Debug, Error)]
+    pub enum FetchError {
+        #[error("Fetch limit {limit} exceeded, fetched: {amount_fetched} from {remote}")]
+        FetchLimitExceeded {
+            limit: usize,
+            amount_fetched: usize,
+            remote: PeerId,
+            fetch_specs: Fetchspecs<PeerId, Revision>,
```

Comment:
```suggestion
            fetchspecs: Fetchspecs<PeerId, Revision>,
```
nit: To keep in line with the naming elsewhere :)

Reply:
Fixed